src/flows/components/ExportTaskOverlay/QueryTextPreview.tsx

```diff
@@ -0,0 +1,39 @@
+import React, {FC} from 'react'
+import {Form} from '@influxdata/clockface'
+import MonacoEditor from 'react-monaco-editor'
+import THEME_NAME from 'src/external/monaco.toml.theme'
+import TOMLLANGID from 'src/external/monaco.toml.syntax'
```

Comment:
oh.. this isn't what i meant when i linked to the toml editor. Add a readonly prop to the flux editor component so we can use that